change/@ni-jasmine-parameterized-b84dc27d-fa72-4663-9049-3d8ddf8c7deb.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
guessing for the [build failure](https://github.com/ni/nimble/actions/runs/9421975523/job/25957227172?pr=2174#step:17:87) that the tgz here does not include the recent changes merged in styleguide

```
[lint:sequential]   208:53  error  Attribute "pattern" has no corresponding i18n attribute. See more at https://angular.io/guide/i18n-common-prepare#mark-element-attributes-for-translations               @angular-eslint/template/i18n
[lint:sequential]   209:59  error  Attribute "display-name" has no corresponding i18n attribute. See more at https://angular.io/guide/i18n-common-prepare#mark-element-attributes-for-translations          @angular-eslint/template/i18n
[lint:sequential]   210:59  error  Attribute "display-name" has no corresponding i18n attribute. See more at https://angular.io/guide/i18n-common-prepare#mark-element-attributes-for-translations          @angular-eslint/template/i18n
[lint:sequential]   220:53  error  Attribute "pattern" has no corresponding i18n attribute. See more at https://angular.io/guide/i18n-common-prepare#mark-element-attributes-for-translations               @angular-eslint/template/i18n
[lint:sequential]   221:59  error  Attribute "display-name" has no corresponding i18n attribute. See more at https://angular.io/guide/i18n-common-prepare#mark-element-attributes-for-translations          @angular-eslint/template/i18n
[lint:sequential]   222:59  error  Attribute "display-name" has no corresponding i18n attribute. See more at https://angular.io/guide/i18n-common-prepare#mark-element-attributes-for-translations          @angular-eslint/template/i18n
[lint:sequential]   242:67  error  Attribute "parent-id-field-name" has no corresponding i18n attribute. See more at https://angular.io/guide/i18n-common-prepare#mark-element-attributes-for-translations  @angular-eslint/template/i18n
[lint:sequential]   255:21  error  Attribute "href-field-name" has no corresponding i18n attribute. See more at https://angular.io/guide/i18n-common-prepare#mark-element-attributes-for-translations       @angular-eslint/template/i18n
```

Those were already handled in https://github.com/ni/javascript-styleguide/pull/146